apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -867,20 +870,62 @@ async fn router(
     let route_split = route.to_str().unwrap().split('/').collect::<Vec<&str>>();
     let res = match (configured_producer, req.method(), &route_split[..]) {
         (Some(configured_producer), &hyper::Method::POST, ["ingest", _]) => {
-            ingest_route(
-                req,
-                // without explicit version, go to current project version
-                if !project.features.data_model_v2 {
-                    route.join(current_version)
-                } else {
-                    route
-                },
-                configured_producer,
-                route_table,
-                is_prod,
-                jwt_config,
-            )
-            .await
+            if project.features.data_model_v2 {
+                // For v2, find the latest version if no version specified
```

Comment:
In dmv1, we had a behavior where hitting `ingest/foo` takes you to `ingest/foo/0.0`.

Now that version is per dmv2 object, when the user hits `ingest/foo`, I try to search for the latest version. If none are found, hit `ingest/foo` directly. Lmk if that's not the behavior we want.